sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -330,4 +338,109 @@ public class ChatClient {
 
         signalingClient!.off(event: event)
     }
+
+    /// Start push notifications. Receiving of notifications can be expected ~1 second after successfully registering.
+    /// - Parameters:
+    ///   - deviceToken: APNS push token.
+    ///   - completionHandler: Success indicates request to register for notifications has been received.
+    public func startPushNotifications(
+        deviceToken: String,
+        completionHandler: @escaping (Result<HTTPResponse?, AzureError>) -> Void
+    ) {
+        guard !pushNotificationsStarted else {
+            completionHandler(.failure(AzureError.client("Push notifications already started.")))
+            return
```

Comment:
Questionable whether you want this to be an error. If you say "startNotifications" and they have already been started, a logical option would be to treat that as success. I'm not saying this has to change, but as error here seems... at least worthy of questioning.

Reply:
It's worth considering whether RT notifications should be updated accordingly. But in a separate PR :)